File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = []
    lst = list(s.split(" "))
    
    
    for el in lst:
        if el and el[0].isdigit(): 
            tmp = el[0] + el[1:].lower()
            answer.append(tmp)
        elif el and not el[0].isdigit() :
            tmp = el[0].upper() + el[1:].lower()
            answer.append(tmp)
        else : 
            answer.append("")
    
    return " ".join(answer)

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = '';
    
    const JadenCase = s.split(" ").map((str,idx) => {
        const tmp = str.split("").map((s,i) => {
            return i !== 0 ? s.toLowerCase() : !Number(s) ? s.toUpperCase() : s })
        return tmp.join("")
    })
    
    return JadenCase.join(" ");
}